spring-petclinic-api-gateway/src/main/java/org/springframework/samples/petclinic/api/ApiGatewayApplication.java

```diff
@@ -15,20 +15,26 @@
  */
 package org.springframework.samples.petclinic.api;
 
+import com.netflix.zuul.ZuulFilter;
+import com.netflix.zuul.context.RequestContext;
 import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.EnableAutoConfiguration;
 import org.springframework.boot.autoconfigure.SpringBootApplication;
 import org.springframework.cloud.client.discovery.EnableDiscoveryClient;
 import org.springframework.cloud.client.loadbalancer.LoadBalanced;
 import org.springframework.cloud.netflix.zuul.EnableZuulProxy;
 import org.springframework.context.annotation.Bean;
 import org.springframework.web.client.RestTemplate;
 
+import java.util.List;
+
 /**
  * @author Maciej Szarlinski
  */
 @EnableZuulProxy
 @EnableDiscoveryClient
 @SpringBootApplication
+@EnableAutoConfiguration
```

Comment:
I think it is redundant due to use of ‘@SpringBootApplication’

Reply:
Sorry for being late to the comment party. ;)
Yeah i agree separate branch.